torch/csrc/distributed/c10d/ProcessGroupXCCL.cpp

```diff
@@ -559,10 +550,9 @@ c10::intrusive_ptr<Work> ProcessGroupXCCL::collective(
   for (const auto& input : inputs) {
     c10::xpu::XPUCachingAllocator::recordStream(
         input.storage().data_ptr(), stream);
+    fn(inputs[i], outputs[i], attr, *comm, stream)
```

Comment:
for (const auto i : c10::irange(inputs.size())) {
    c10::xpu::XPUCachingAllocator::recordStream(
        inputs[i].storage().data_ptr(), stream);
    fn(inputs[i], outputs[i], attr, *comm, stream);
  }

Reply:
I think we don't need attr as well. Let me change here.